README.md

```diff
@@ -1 +1,101 @@
 # SpatialConnect
+
+#### Fresh install of EFC and the associated dependencies. ####
+
+#### **Dependencies List** ####
+What you will need to have installed prior to running the program:
+
+##### Git repositories #####
+* [spatialconnect-mobile](https://github.com/boundlessgeo/spatialconnect-mobile)
+* [React-native-maps](https://github.com/boundlessgeo/react-native-maps)
+* [React-native-spatialconnect](https://github.com/boundlessgeo/react-native-spatialconnect)
+
+##### Signing apps #####
+
+First you will need to obtain the signing key file and passwords from one of the project
+leaders.  Email spatialconnect@boundlessgeo.com to request it.
+
+
+##### General #####
+
+
+* Homebrew:
+``
+/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+``
+
+* Node:
+``
+brew install node
+``
+
+* Watchman:
+``
+brew install watchman
+``
+
+* React-native CLI:
+``
+npm install -g react-native-cli
+``
+
+* Flow:
+``
+npm i flow-bin@ --save-dev
+``
+
+
+###### iOS specific ######
+* Xcode
+* Carthage
+* Fabric
+* Crashlytics.framework
+
+###### Android specific ######
+* Android Studio
+* Java SE Runtime 1.7
+* Java Development Kit 1.8
+* gradle.properties
+
+#### iOS setup ####
+* Install Xcode from the Apple App store.
+* Install Carthage
+``
+npm install carthage
+``.
+* Make sure all three git repositories are on a common level in your directory structure.
+* Add the Crashlytics.framework file into the _spatialconnect-mobile_ > _ios_ folder.
+* In the _spatialconnect-mobile folder_
+``
+npm i
+``.
+* Next
+``
+npm i react-native-spatialconnect
+``.
+* Finally **either**
+``
+react-native run-ios
+``
+**or** in the _ios_ folder of _spatialconnect-mobile_ open the SCMobile.xcodeproj file.
+
+#### Android setup ####
+* Install [Android Studio](https://developer.android.com/studio/index.html)
+* Add the gradle.properties file to _spatialconnect-mobile_ > _android_ folder.
+* Next
+``
+npm i
+``.
+* In the _spatialconnect-mobile folder_
+``
+npm i
+``.
+* Next
+``
+npm i react-native-spatialconnect
```

Comment:
This command shouldn't be needed if you previously ran `npm i`, which installs all the dependencies, including react-native-spatialconnect.